src/cogito/package.d

```diff
@@ -36,6 +36,30 @@ private Result runOnFile(string file)
     return typeof(return)(visitor.source);
 }
 
+/**
+ * Map a filename to an array of files.
+ *
+ * Params:
+ *     filename = File path.
+ *
+ * Returns: Array of filenames.
+ *   If the filename points to a folder all d-files are returned,
+ *   If the filename points to a file an array with only that file is returned.
+ */
+string[] toFiles(string filename)
+{
+    import std.file : isFile, isDir, dirEntries, SpanMode;
+    if (filename.isFile)
+    {
+        return [filename];
+    }
+    if (filename.isDir)
+    {
+        return filename.dirEntries("*.d", SpanMode.breadth).filter!("a.isFile").map!("a.name").array;
+    }
+    assert(false);
```

Comment:
So, we have
```
$ build/release/bin/cogito /dev/null
Illegal instruction (core dumped)
```

I would prefer
```
throw new Exception(format!"%s is neither a file nor a directory", name);
```
in this case.

Especially if exceptions are caught in some place.

Reply:
I added an exception handler into the main function.